KevBeltrao/gelada-mobile
# Feature ✨ theme provider

The PR meets the following points:

- [x] The commit follows the commit style guide




# What I did

## ✨ New features

- #1  **Create pull request template for the next tasks**
  - [feautre(git) ✨ create pull request template](https://github.com/KevBeltrao/gelada-mobile/commit/63e3da74f6ac1d6970d1565a12272cba4b257bdb)
- #1 **Create theme provider for the next tasks**
  -  [feautre(git) ✨ create theme provider on app](https://github.com/KevBeltrao/gelada-mobile/commit/25def02ac4c267251066cddaad78520cfd3d3cb4)
  - [feature(theme) 🔨 improve theme provider and change homepage](https://github.com/KevBeltrao/gelada-mobile/commit/4aad8c50cbd3813d9719965cee1bae4e7b3701f1)



# How to test

1. run `yarn` to install all depedencies
2. run `yarn web` to run the app
3. scan the qr code and open in on expo app
4. see if the theme is correct acording to the figma
